### PR TITLE
Update README introduction via PyMC repo

### PR DESCRIPTION
**Pull request by CloudChaoszero**
Update README introduction via PyMC repo 
* "Copy Paste"ish from the PyMC3 README.rst repo
* Remove README.md file
* Also add one item in .gitignore

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+Papers citing PyMC3
+-------------------
+
+See `Google Scholar <https://scholar.google.de/scholar?oi=bibs&hl=en&authuser=1&cites=6936955228135731011>`__ for a continuously updated list.
+
+Contributors
+============
+
+See the `GitHub contributor
+page <https://github.com/pymc-devs/pymc3/graphs/contributors>`__. Also read our `Code of Conduct <https://github.com/pymc-devs/pymc3/blob/master/CODE_OF_CONDUCT.md>`__ guidelines for a better contributing experience.
```
Comment (OriolAbril) [inline README.rst:65]: The link should be to the contributor page for pymc-examples, code of conduct is fine like this for now

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+
+Contact
+=======
+
+We are using `discourse.pymc.io <https://discourse.pymc.io/>`__ as our main communication channel. You can also follow us on `Twitter @pymc_devs <https://twitter.com/pymc_devs>`__ for updates and other announcements.
+
+To ask a question regarding modeling or usage of PyMC3 we encourage posting to our Discourse forum under the `“Questions” Category <https://discourse.pymc.io/c/questions>`__. You can also suggest feature in the `“Development” Category <https://discourse.pymc.io/c/development>`__.
+
+To report an issue with PyMC3 please use the `issue tracker <https://github.com/pymc-devs/pymc3/issues>`__.
```
Comment (OriolAbril) [inline README.rst:75]: Here we should probably keep this link and add that issues related to examples notebooks should be posted here + add the link to issue tracker of pymc-examples

Comment (CloudChaoszero) [reply README.rst]: Sounds good. I added the updates links & kept the existing one.

**Review by OriolAbril — APPROVED** [review]

**Comment by fonnesbeck** [comment]
@twiecki Do we still need to include Quantopian as a listed sponsor?

**Comment by OriolAbril** [comment]
> @twiecki Do we still need to include Quantopian as a listed sponsor?

and while we are at this, I think the whole governance doc needs an update, and in my opinion some extensions too.

**Comment by fonnesbeck** [comment]
@OriolAbril please do. We pulled that together when we first joined NumFOCUS and it has not been updated since.

**Pull request merged**